gradle-plugin/src/main/groovy/akka/grpc/gradle/AkkaGrpcPluginExtension.groovy

```diff
@@ -16,21 +16,26 @@ class AkkaGrpcPluginExtension {
     // workaround for tests, where there's no jar and MANIFES.MF can't be read
     final String pluginVersion = System.getProperty("akkaGrpcTest.pluginVersion", AkkaGrpcPlugin.class.package.implementationVersion)
 
-    String language
     boolean generateClient = true
     boolean generateServer = true
     boolean generatePlay = false
     boolean serverPowerApis = false
     boolean usePlayActions = false
     List<String> extraGenerators = []
 
+    private final Project project
+
+    final boolean scala
```

Comment:
Ah, so this is not a plugin option but just an internal variable, and as soon as 1 .scala source file appears the project switches to 'scala' mode. I guess that's OK for most common cases.